packages/site/src/components/ComponentLinks.tsx

```diff
@@ -9,16 +10,68 @@ import { useAtlantisSite } from "../providers/AtlantisSiteProvider";
  */
 export const ComponentLinks = ({
   links,
+  goToProps,
+  goToUsage,
+  goToDesign,
 }: {
   readonly links?: ContentExportLinks[];
+  readonly goToProps: () => void;
+  readonly goToUsage: () => void;
+  readonly goToDesign: () => void;
 }) => {
+  const [hlinks, setHlinks] = useState<Element[] | null>(null);
+
+  useEffect(() => {
+    const hdd = document.querySelectorAll("[data-heading-link]");
+
+    if (hdd.length > 0) {
+      setHlinks(Array.from(hdd));
+    }
+  }, []);
+
+  const click = (e: MouseEvent) => {
+    e.preventDefault();
+    const id = e.currentTarget?.getAttribute("href")?.replace("#", "");
+
+    if (id) {
+      goToDesign();
+      setTimeout(() => {
+        const element = document.getElementById(id);
+
+        if (element) {
+          element.scrollIntoView({ behavior: "smooth" });
+        }
+      }, 100);
+    }
+  };
   const { isMinimal } = useAtlantisSite();
   if (isMinimal) return null;
 
   return (
     <Content>
       <Box padding="base" direction="column">
-        <Heading level={2}>Links</Heading>
+        <Box margin={{ bottom: "base" }}>
+          <Heading level={3}>Design</Heading>
```

Comment:
[To match Figma](https://www.figma.com/design/HhNrPoreO9D8R8O9Ta2AM7/Unified-Atlantis-Docs?node-id=1333-20309&t=TCOMi34VXaWhjAec-1), should be using 6
```suggestion
          <Heading level={6} element={h3}>Design</Heading>
```